Formula/commandbox.rb

```diff
@@ -1,16 +1,16 @@
 class Commandbox < Formula
   desc "CFML embedded server, package manager, and app scaffolding tools"
   homepage "https://www.ortussolutions.com/products/commandbox"
-  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/5.0.1/commandbox-bin-5.0.1.zip"
-  sha256 "9f855c5b796ac36b010d8b64d58664650d381446d52e0e9167249d4df0269215"
+  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/5.1.0/commandbox-bin-5.1.0.zip"
+  sha256 "cd5a903cc4874930bfd9437e46240f7a980ff7f28c1e136299dc525c47d5ff28"
 
   bottle :unneeded
 
   depends_on :java => "1.8+"
```

Comment:
```suggestion
  depends_on "openjdk"
```

Reply:
Since java 1.7 is not actually compatible, only 1.8+ is this suggestion moot?

Reply:
openjdk will always just be the latest java we can provide, which should be used if possible.

Reply:
Fair enough.  I'll change this now.